src/compiler/core.ts

```diff
@@ -895,6 +895,14 @@ namespace ts {
         return result;
     }
 
+    export function arrayFromMap<T, U>(iterator: Iterator<T>, f: (value: T) => U) {

```

Comment:
I meant map in the sense of the `map` function not the `Map` type. Maybe `mapIterator`? `mapIteratorToArray`? Something else?